src/test/java/com/checkout/payments/RequestPaymentsTestIT.java

```diff
@@ -84,6 +106,13 @@ void shouldMakeCardPayment() {
         assertNotNull(paymentResponse);
         assertNotNull(paymentResponse.getId());
         assertEquals(Long.valueOf(10), paymentResponse.getAmount());
+//        assertEquals(paymentResponse.getPaymentType(), PaymentType.RECURRING);
```

Comment:
Remove commented code

Reply:
Now, this response don't return these properties, I think comment these possibilities is good example for the integrators.